app/src/views/teachers/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import { useRouter } from "next/router";
+import Head from "next/head";
+
+import Breadcrumbs from "@/components/Breadcrumbs";
+import Title from "@/components/Title";
+import Text from "@/components/Text";
+import Dropdown from "@/components/Dropdown";
+
+import { DropdownOption } from "@/types/Generic";
+
+import { teachersContent } from "@/constants/content";
+import { teachersVideos } from "@/constants/teachersVideos";
+
+const TeachersPage = () => {
+  const [selectedOption, setSelectedOption] = useState({
+    value: teachersContent.defaultVideo.value,
+    name: teachersContent.defaultVideo.name,
+  });
+  const [selectedVideo, setSelectedVideo] = useState(teachersContent.defaultVideo.url);
+
+  const router = useRouter();
+
+  const handleBackClick = () => {
+    router.push("/");
+  };
+
+  const handleVideoChange = ({ value, name }: DropdownOption) => {
+    setSelectedOption({ value, name });
+    const selectedVideoOption = teachersVideos.find((video) => video.name === name);
+    setSelectedVideo(selectedVideoOption!.url);
```

Comment:
What does "!" mean? :) Just curious, maybe you wanted to use "?" here?

Reply:
I am telling typescript explicitly that this can't be undefined because it is a constant